src/test/asyncIteratorSubscription.ts

```diff
@@ -171,3 +171,36 @@ describe('GraphQL-JS asyncIterator', () => {
     return r;
   });
 });
+
+function isEven(x: number) {
+    if (x === undefined) {
```

Comment:
Following the same indentation rules as the rest of the file (2 spaces) would help with readability.

Reply:
Sorry, forgot to run the formatter